src/main/java/io/spaship/operator/service/k8s/CommandExecutionService.java

```diff
@@ -167,7 +167,11 @@ private CommandExecutionOutput deleteTarget(Pod targetPod, String target) throws
                 var output = executeCommandInContainer(targetPod, command);
                 yield new CommandExecutionOutput(targetPod.getMetadata().getName(), output);
             }
-            case UNKNOWN -> throw new IllegalArgumentException("Invalid target type: UNKNOWN");
+            case UNKNOWN -> {
```

Comment:
Hi @arkaprovob , 

I've made the changes accordingly. Added `BROKEN_SYMLINK` as the new target type. If we'll get any broken symlink it will update that and we'll keep the `UNKNOWN` block as same.

```
[ -L " + target + " ] checks if the target is a symbolic link.
! [ -e " + target + " ] checks if the symbolic link is broken, i.e., the target it points to doesn't exist (-e checks if the file exists).
```